docs/real_world_fl/overview.rst

```diff
@@ -70,42 +70,9 @@ name in the workspace folder created where provision.py is run.
    signed by :class:`SignatureBuilder<nvflare.lighter.impl.signature.SignatureBuilder>` so the system will detect if any
    of the files have been altered and may not run.
 
```

Comment:
Where are the instructions now on how to generate password protected startup kits?

Reply:
I will need to add a section for that